source/Launchpad-Applications-Tests/CurrentlyRunningLaunchpadApplicationTest.class.st

```diff
@@ -0,0 +1,60 @@
+Class {
+	#name : 'CurrentlyRunningLaunchpadApplicationTest',
+	#superclass : 'TestCase',
+	#instVars : [
+		'currentlyRunningLaunchpadApp'
+	],
+	#category : 'Launchpad-Applications-Tests',
+	#package : 'Launchpad-Applications-Tests'
+}
+
+{ #category : 'running' }
+CurrentlyRunningLaunchpadApplicationTest >> setUp [
+
+	super setUp.
+	CurrentlyRunningLaunchpadApplication initialize.
```

Comment:
This isn't necessary. Class initialize methods are sent during loading.

Reply:
Addressed in 457319a